src/lib/pickles/step_verifier.ml

```diff
@@ -461,6 +461,12 @@ struct
         let sample () = squeeze_challenge sponge in
         let sample_scalar () = squeeze_scalar sponge in
         let open Plonk_types.Messages.In_circuit in
+        let without = Type.Without_degree_bound in
+        let absorb_g gs = absorb sponge without gs in
+        let sg_old : (_, Wrap_hack.Padded_length.n) Vector.t =
+          Wrap_hack.Checked.pad_commitments sg_old
```

Comment:
I suggest a comment on what's happening here. IIUC we're adding dummy `sg` in this vector to make it of size 2. I'm guessing it is because the maximum number of proofs we're verifying is 2. The generation of the dummy is puzzling though, why deterministically generate them from a random value instead of using something like `(0, 0)`?

Reply:
probably a good idea to rename `sg_old` here to reflect that it's a vector. Perhaps `prev_commits`